### DISCUSSION TITLE
Unit redeemer

### DISCUSSION THREAD

**Discussion by ajuggler** [Python-Cardano/pycardano#248] [Q&A] [answered]
I am building a transaction which contains a minting script `mint_script` which assumes a *unit* redeemer.  If I use either

```python
tx_builder.add_minting_script(mint_script)
```
or
```python
tx_builder.add_minting_script(mint_script, redeemer=None)
```
I get the error
```shell
AttributeError: 'NoneType' object has no attribute 'index'
```
which seems to indicate that I need to explicitly pass a unit redeemer.

I tried
```python
tx_builder.add_minting_script(mint_script, redeemer=Redeemer(None))
```
but that did not work.  How do I pass a *unit* redeemer?

**Comment by cffls** [comment, marked as answer]
Maybe try `Redeemer(PlutusData())` or `Redeemer(RawCBOR(bytes.fromhex("d87a80")))`

**Reply by ajuggler** [reply]
`Redeemer(PlutusData())` seems to work fine.  Thanks!
